internal/pkg/cli/validate.go

```diff
@@ -110,6 +116,35 @@ func validateSvcType(val interface{}) error {
 	return fmt.Errorf("invalid service type %s: must be one of %s", svcType, prettify(manifest.ServiceTypes))
 }
 
+func validateJobName(val interface{}) error {
+	if err := basicNameValidation(val); err != nil {
+		return fmt.Errorf("job name %v is invalid: %w", val, err)
+	}
+	return nil
+}
+
+func validateJobSchedule(sched string) error {
+	cronErr := basicCronValidation(sched)
+	if cronErr == nil {
+		return nil
+	}
+	timeErr := basicDurationValidation(sched, 60)
+	if timeErr == nil {
+		return nil
+	}
+	if cronErr != timeErr {
+		return fmt.Errorf("cron: %w; duration: %v", cronErr, timeErr)
+	}
+	return fmt.Errorf("%s: %w", errScheduleInvalid, cronErr)
```

Comment:
Can we add a comment explaining what's going on here? It seems like this case will happen if the customer specified an `@every` definition string whose rate is too high (more than every minute). 

Maybe since the cron validation is more complex, we can do the simple duration string validation first and lazily return if it's ok? That way if there's an error there, we'll know we have to try a more complex validation, and we may not even have to handle this extra error case where we test. We can return more descriptively based on the cron error, and then finally return `errScheduleInvalid` without wrapping any other errors if no other validator worked and we can't give a more descriptive error.

Reply:
I may be misunderstanding, but won't the above change lose the details of duration errors? All rate duration errors like "50 sec" or ".5s" will get the generic `errScheduleInvalid` error, right? 

(Yes, exactly--that case is for `@every` + poorly formed/invalid duration like `@every 5min` or `@every 30s`. I'll add a comment! :) )

Reply:
As per offline discussion: I refactored this to no longer accept durations for schedules, besides those intervals/rates accepted by cron (preceded by `@every`). `1h30m` is no longer valid, but `@every 1h30m` still is. Adjusted tests accordingly.